microsoft-edge/web-platform/site-impacting-changes.md

```diff
@@ -23,7 +23,8 @@ The web is constantly evolving to improve the user experience, security, and pri
 | Deprecate AppCache | [Chrome+1](#release-comments) \(Edge v86\)  |  | This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, navigate to the [WebDev documentation][WebDevAppCacheRemoval].  The Microsoft rollout schedule for deprecation is planned for one release after Chrome.  Requesting an [AppCache OriginTrial Token][ChromeDevelopersOrigintrialsAppCacheOriginTrial] allows sites to continue to use the deprecated API until Edge v90.  |  
 | Removal of Adobe Flash | Edge v88  |  | This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, navigate to the [Adobe Flash Chromium Roadmap][ChromiumFlashRoadmapSupportRemoved].  | 
 | Turn off and remove FTP | Edge v88  | Edge Beta v87 | In Edge Beta v87, FTP support is turned off by default; in Edge Stable v87 it remains enabled.  In Edge v88, FTP support is removed entirely.  This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, navigate to the [Chrome Platform Status Entry][ChromePlatformStatus6246151319715840].  Enterprises that have sites that still require FTP support can continue to use FTP by configuring the site to use [IE mode][DeployedgeEdgeIeMode].  | 
-| Autoupgrade mixed content images | Edge v88  |  | Non-secure (HTTP) references to images are automatically upgraded to HTTPS; if the image is not available over HTTPS, the image download fails. A [Group Policy][DeployedgePoliciesInsecurecontentallowedforurls] is available to control this feature. This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, navigate to the [Chrome Platform Status entry][ChromePlatformStatus4926989725073408].  |  
+| Autoupgrade mixed content images | Edge v88  |  | Non-secure (HTTP) references to images are automatically upgraded to HTTPS; if the image is not available over HTTPS, the image download fails. A [Group Policy][DeployedgePoliciesInsecurecontentallowedforurls] is available to control this feature. This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, navigate to the [Chrome Platform Status entry][ChromePlatformStatus4926989725073408].  | | HTTP authentication disallowed when third party cookies are blocked  | Edge v87  |  | Starting with Microsoft Edge version 87, when cookies are blocked for third party requests using the [BlockThirdPartyCookies](https://docs.microsoft.com/deployedge/microsoft-edge-policies#blockthirdpartycookies) policy, HTTP authentication is also disallowed. In some cases, this will impact Internet Explorer mode causing site list download to fail. You can allow cookies for the specific domains hosting your Enterprise Mode Site List using the [CookiesAllowedForURLs](https://docs.microsoft.com/en-us/deployedge/microsoft-edge-policies#cookiesallowedforurls) policy to ensure that site list downloads are successful.  |   
```

Comment:
Tweaked text; can you use this instead?:

Starting with Edge v87, when cookies are blocked for third party requests, either via the [BlockThirdPartyCookies](https://docs.microsoft.com/deployedge/microsoft-edge-policies#blockthirdpartycookies) policy or via the Edge Settings page, HTTP authentication is also disallowed. This can impact [Enterprise Mode IE website list](https://docs.microsoft.com/en-us/deployedge/edge-ie-mode-policies#configure-using-the-use-the-enterprise-mode-ie-website-list-policy) downloads if the endpoint hosting the list requires the use of HTTP authentication. You can allow the use of both cookies and HTTP authentication for Enterprise Mode Site List downloads by adding a matching URL pattern to the [CookiesAllowedForURLs](https://docs.microsoft.com/en-us/deployedge/microsoft-edge-policies#cookiesallowedforurls) policy."

Reply:
Thanks for the review @erik-anderson. Updated text.

Reply:
Resolving conversation.